### PR TITLE
Completed the TBD examples in README file

### PR DESCRIPTION
**Pull request by omnone**
I've added the missing examples (TBD) from readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,124 +258,127 @@ var spotifyApi = new SpotifyWebApi();
 
 // Get multiple albums
 spotifyApi.getAlbums(['5U4W9E5WsYb2jUQWePT8Xm', '3KyVcddATClQKIdtaap4bV'])
-  .then(function(data) {
-    console.log('Albums information', data.body);
-  }, function(err) {
-    console.error(err);
-  });
+    .then(function (data) {
```
Comment (thelinmichael) [inline README.md:261]: Seems like you've got different style settings than the code in this repository, e.g. 4 spaces per tab, and space in `function (`. There's no right and wrong here, but in order to be consistent the style needs to stay the same as it is.

**Comment by thelinmichael** [comment]
Thanks a lot @omnone! Looking forward to merging this.

**Comment by omnone** [comment]
@thelinmichael I think now it's ok , please take a look.

**Comment by thelinmichael** [comment]
Great stuff! Thanks @omno

**Comment by omnone** [comment]
Hey @thelinmichael because I am participating in Hacktoberfest,  can you add "hacktoberfest-accepted" label to the PR ? Thanks!

**Comment by thelinmichael** [comment]
@omnone Done!

**Pull request merged**